### DISCUSSION TITLE
Tabs responsive con css

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#24] [Announcements]
# Tabs responsive con css

Pestañas de navegación responsive usando solo css, fáciles de usar y personalizar

https://www.zkreations.com/2023/10/tabs-responsive-con-css.html

**Comment by TakoMics** [comment]
Estas tabs me van geniales para un proyecto que tengo en mente.

Como siempre, Daniel, una explicación genial y un resultado fantástico. Me encanta la nueva estructura limpia y más sencilla de ententer que manejas.

Un maestro. Mil gracias.

**Reply by danieIabel** [reply]
Gracias por pasar amigo, aprecio mucho sus comentarios

**Comment by ivisjose** [comment]
hola, amigo, tengo versión de plantilla Bison premium, no habría problemas al instalarlo en mi plantilla ya que la interna no me gusta :D

**Reply by danieIabel** [reply]
Buenas amigo, es un componente aparte asi que deberia de poder instalarlo sin problemas. Desarrolle esto como componente independiente ya que pretendo eliminar este shortcode de la próxima version de todos mis temas.

**Reply by ivisjose** [reply]
muchas gracias amigo ya lo instale y trabajando al 110%, esperando próxima actualización :P

**Reply by danieIabel** [reply]
Me alegra de que te haya servido amigo. Un saludo.

![gif](https://media.giphy.com/media/ptqAPgghLtHOa0SLJS/giphy.gif)
